TrinityAudioDemo/PulseSlidingViewController.swift

```diff
@@ -14,6 +14,10 @@ class PulseSlidingViewController: UIViewController {
     @IBOutlet var triniyPlayerView: TrinityPlayerView!
     @IBOutlet var loadingIndicator: UIActivityIndicatorView!
     @IBOutlet weak var playerIdLb: UILabel!
+    @IBOutlet weak var playerHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var playerTopConstraint: NSLayoutConstraint!
+    
+    var autoPlay = false
```

Comment:
May be we should set it  to true (like our only consumer will use case)?

Reply:
@ilyaa-trinityaudio 

yes, I set it to be "true" by default https://github.com/TrinityAudioSDK/trinityaudio-ios-demo/pull/11/commits/ba65386e075a085284e0b23bf874a3878d29b7af